tests/integrations/celery/test_celery.py

```diff
@@ -42,6 +42,7 @@ def inner(propagate_traces=True, backend="always_eager", **kwargs):
 
             # this backend requires capture_events_forksafe
             celery.conf.worker_max_tasks_per_child = 1
+            celery.conf.worker_concurrency = 1
```

Comment:
This speeds up tests, otherwise celery forks 20 times.

Reply:
Sounds good on the perspective of speeding up tests -- won't this have an effect on bugs and code paths surfaced by tests, though?

Reply:
So we haven't had any bugs related to this *so far* and my gut feeling tells me no. There is no direct communication between the forked processes so I'd say it's unlikely, and the tests won't send in high event volumes ever anyway.